src/redux/actions/brownbagActions.js

```diff
@@ -90,15 +90,21 @@ export function getNextPresenter(presenter) {
 }
```

Comment:
This file has errors in that you have not refactored all the name changes you made to the actions and action types. Who's on the list is not loading as the action referenced in the component is not available

Reply:
Let me fix that